README.md

```diff
@@ -1,40 +1,22 @@
 # Pa11y CI
 
-Pa11y CI is an accessibility test runner built using [Pa11y] focused on running on Continuous Integration environments.
-
-Pa11y CI runs accessibility tests against multiple URLs and reports on any issues. This is best used during automated testing of your application and can act as a gatekeeper to stop a11y issues from making it to live.
-
 [![NPM version][shield-npm]][info-npm]
 [![Node.js version support][shield-node]][info-node]
 [![Build status][shield-build]][info-build]
-[![Dependencies][shield-dependencies]][info-dependencies]
 [![LGPL-3.0-only licensed][shield-license]][info-license]
 
----
-
-## Table of contents
-
-* [Table of contents](#table-of-contents)
-* [Requirements](#requirements)
-* [Usage](#usage)
-  * [Configuration](#configuration)
-  * [Default configuration](#default-configuration)
-  * [URL configuration](#url-configuration)
-  * [Sitemaps](#sitemaps)
-* [Reporters](#reporters)
-  * [Use Multiple reporters](#use-multiple-reporters)
-  * [Reporter options](#reporter-options)
-  * [Write a custom reporter](#write-a-custom-reporter)
-    * [Configurable reporters](#configurable-reporters)
-  * [Docker](#docker)
-* [Tutorials and articles](#tutorials-and-articles)
-* [Contributing](#contributing)
-* [Support and Migration](#support-and-migration)
-* [Licence](#licence)
+Pa11y CI is an accessibility test runner built using [Pa11y], designed to run in Continuous Integration environments. Automated testing of your application can help to prevent accessibility issues reaching production.
+
+Use this tool to test against a list of URLs or a sitemap, and report on issues it finds. 
+
+> [!NOTE]  
+> Version 3 of Pa11y CI requires a tweak to make it work with versions of Ubuntu above 20.04.  Version 4 will be unaffected by this issue.
```

Comment:
How mysterious! 😄  Do we have an issue we can link-to describing this tweak?

Reply:
I've updated the note to describe one fix, contributed by @aabccd021 (thanks!)